tests/common.go

```diff
@@ -4397,6 +4399,61 @@ func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
 	}
 }
 
+func collectAndCopyLogsForTest(testCaseName string, podLabel map[string]string, namespaceFunc func() (string, error), logLabel string) {
+	testCaseName = strings.ReplaceAll(testCaseName, " ", "")
+	namespace, err := namespaceFunc()
+	if err != nil {
+		log.Errorf("Error in getting namespace for the [%s] logs of test case [%s], Err: %v", logLabel, testCaseName, err.Error())
+		return
+	}
+	podList, err := core.Instance().GetPods(namespace, podLabel)
+	if err != nil {
+		log.Errorf("Error in getting pods for the [%s] logs of test case [%s], Err: %v", logLabel, testCaseName, err.Error())
+		return
+	}
+	masterNode := node.GetMasterNodes()[0]
+	err = runCmd("pwd", masterNode)
```

Comment:
Why are we doing this?

Reply:
In case of a failure of the runCmd function, it returns an error message that includes the command it ran. I am assuming, pwd might be more useful for de-bugging. This convention is taken from the legacy code.

Reply:
ok